oxidation/libparsec/crates/client_types/src/lib.rs

```diff
@@ -5,6 +5,7 @@ mod local_device;
 mod local_device_file;
 mod local_manifest;
 mod path;
+mod test;
```

Comment:
```suggestion
#[cfg(test)]
mod test;
```
We only want to include this module during test

Reply:
Done.